README.md

```diff
@@ -31,12 +31,11 @@ development and testing purposes.
 * [Git](https://git-scm.com/downloads)
 * [Node.js](https://nodejs.org/en/download/)
 * [Yarn](https://yarnpkg.com/en/docs/install)
-* `yarn --add-python-to-path install --global --production windows-build-tools` (Windows only)
 
 ### One-time Setup
 
 1. Clone this repo
-2. `DEPS=true ./build.sh`
+2. `./build.sh`
```

Comment:
why did you get rid of DEPS=true?

Reply:
There's no need anymore for installing dependencies from the build script. The user should install the prerequisites listed in `README.md`. That is Git, Node, and Yarn. No need anymore for Python for developing.

Reply:
it would be much better if users ran a script to install dependencies instead of following a README. that way if the dependencies change, no one has to remember to update the README